src/views/Swap/Swap.vue

```diff
@@ -211,7 +211,7 @@
               class="btn btn-primary btn-lg"
               id="swap_review_button"
               @click="review"
-              :disabled="!canSwap || cannotCoverNetworkFee"
+              :disabled="false"
```

Comment:
@bradleySuira Do we need the prop if we are defaulting it to false?

Reply:
fixed man, thanks!